packages/legacy/core/App/types/security.ts

```diff
@@ -10,11 +10,16 @@ export enum AuthLevel {
   BiometricsOnly = 'BiometricsOnly',
 }
 
+/*
+  no_repeated_numbers - adjacent characters allowed repeating times
+  0 - Disable adjacent number repeating validation, any times repeating are allowed
+  n > 0 - Enable the repeating validation, n repeating times are forbidden, e.g. n = 2 forbid repeating 2 times, '11' is allowed but '111' forbidden
+*/
 export interface PINValidationRules {
   only_numbers: boolean
   min_length: number
   max_length: number
-  no_repeated_numbers: boolean | number
+  no_repeated_numbers: number
   no_repetition_of_the_two_same_numbers: boolean | number
```

Comment:
We should probably make the same change for the rules below:
```suggestion
  no_repeated_numbers: number
  no_repetition_of_the_two_same_numbers: number
```